src/lib/components/create_order/index.js

```diff
@@ -0,0 +1,36 @@
+import _metaphysics from '../../../lib/metaphysics.coffee'
+import query from './mutation'
+
+// FIXME: Rewire
+let metaphysics = _metaphysics
+
+export const createOrder = (
+  user,
```

Comment:
it might be easier to just make these all part of an object that's being passed in. That object can have an interface (and this can be a TS file).